README.md

```diff
@@ -1,76 +1,110 @@
+
+All of these features are available in our open source community edition.
+
+- Deploy **extremely fast** via CDN, npm, CLI, or Docker.
+- Bundle your docs into a **zero-dependency** HTML file (when using [`redoc-cli`]()).
+- Integrate with any React app, with client or server-side rendering.
+- Take full advantage of OpenAPI 2.0/3.0 features, from `discriminator` to `allOf`.
+ <br> <br>
 ![](docs/images/discriminator-demo.gif)
-- OpenAPI 3.0 support
-- Neat **interactive** documentation for nested objects <br>
+- Use **interactive** documentation for nested objects
+ <br> <br>
 ![](docs/images/nested-demo.gif)
-- Code samples support (via vendor extension) <br>
+ <br> <br>
+- Include **code samples** (via vendor extension)
+ <br> <br>
 ![](docs/images/code-samples-demo.gif)
-- Responsive three-panel design with menu/scrolling synchronization
-- Integrate API Introduction into side menu - ReDoc takes advantage of markdown headings from OpenAPI description field. It pulls them into side menu and also supports deep linking.
-- High-level grouping in side-menu via [`x-tagGroups`](docs/redoc-vendor-extensions.md#x-tagGroups) vendor extension
-- Simple integration with `create-react-app` ([sample](https://github.com/APIs-guru/create-react-app-redoc))
-- Branding/customizations via [`theme` option](#redoc-options-object)
+ <br> <br>
+- Get a fully responsive, three-panel design out of the box (with menu/scrolling synchronization)
+- Integrate your API Introduction into the side menu (with support for deep linking)
+- Create high-level groupings in the side menu with the [`x-tagGroups`](docs/redoc-vendor-extensions.md#x-tagGroups) vendor extension
+- Match your docs and your brand by using the [`theme` option](#redoc-options-object)
+
+## Premium Features
+
+These features are availale in our hosted or on-premise premium edition.
+
+- SEO Friendly URLs
+- 10x Page Load Speed
+- 'Try It' API console
+  - Multiple Servers
+  - OAS3 security schemes
+- Enhanced Search
+- Additional Theming Control
+- Embedded Markdown
+
+## Live Examples
+- [Rebilly](https://rebilly-api.redoc.ly/)
+- [Docker Engine](https://docs.docker.com/engine/api/v1.25/)
+- [Zuora](https://www.zuora.com/developer/api-reference/)
+- [Discourse](http://docs.discourse.org)
+- [Commbox](https://www.commbox.io/api/)
+- [APIs.guru](https://apis.guru/api-doc/)
+- [FastAPI](https://github.com/tiangolo/fastapi)
+- [Spoke Phone](https://developer.spokephone.com/)
+- [SignRequest](https://signrequest.com/api/v1/docs/)
 
 ## Roadmap
-  - [x] ~~[OpenAPI v3.0 support](https://github.com/Redocly/redoc/issues/312)~~
+  - [x] ~~OpenAPI v3.0 support~~
   - [x] ~~performance optimizations~~
   - [x] ~~better navigation (menu improvements + search)~~
   - [x] ~~React rewrite~~
   - [x] ~~docs pre-rendering (performance and SEO)~~
-  - [ ] ability to simple branding/styling
-  - [ ] built-in API Console
+  - [ ] use `redoc-cli` under the hood
```

Comment:
Here I think we want to use `openapi-cli` under the hood. Should we also link to the specific issues related to the roadmap items?

Reply:
updated